### DISCUSSION TITLE
The rack SVG cannot display

### DISCUSSION THREAD

**Discussion by zanchuan** [netbox-community/netbox#9690] [General]
![image](https://user-images.githubusercontent.com/25120498/177985312-affac0ca-0b81-491c-a9b3-77ef91559615.png)
Please help solve the problem,thanks

**Comment by julien-flume** [comment]
Hello

I have the same issue, some people solved this issue ?

I'm following this documentation to try to solve it :https://github.com/netbox-community/netbox/wiki/Common-Issues#rack-elevations-do-not-display-v27

The call API is good but when I parse the header, but the option seems to be good as well
![image](https://user-images.githubusercontent.com/85941769/198379674-bd4791af-8a07-4621-8351-0c31166fd566.png)



Regards,

**Comment by luistechnology** [comment]
Did anyone get a response here?

**Reply by candlerb** [reply]
A very similar issue was discussed in #10815 and that will give you some clues as to what to look for.
